File: 3174-minimum-number-of-changes-to-make-binary-string-beautiful/3174-minimum-number-of-changes-to-make-binary-string-beautiful.java
class Solution {
    public int minChanges(String s) {
        int l = s.length(); 

        int count = 0;
        for (int i = 0; i < l - 1; i += 2) { 
            if (s.charAt(i) != s.charAt(i + 1)) { 
                count++;
            }
        }

        return count;
    }
}

File: 3174-minimum-number-of-changes-to-make-binary-string-beautiful/3174-minimum-number-of-changes-to-make-binary-string-beautiful.cpp
class Solution {
public:
    int minChanges(string s) {
        int l = s.size();

        int count =0;
        for(int i=0; i<l-1; i = i+2){
            if(s[i] != s[i+1]){
                count++;
            }

        }

        return count;
    }
};